kubernetes/kubernetes_rce_via_open_access/kubernetes.md

```diff
@@ -0,0 +1,114 @@
+# RCE in Kubernetes Cluster with Open Access
+
+# Prerequisites
+
+1. Install Docker
+
+2. Download and install minikube suitable for required system/arch from:
+
+`https://minikube.sigs.k8s.io/docs/start/`
+
+E.g for Linux amd64 run:
+
+```bash
+curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+sudo install minikube-linux-amd64 /usr/local/bin/minikube`
+```
+
+3. Run the cluster with:
+
+`$ minikube start --driver=docker --force`
+
+4. Create kubectl alias:
```

Comment:
This is an alternative way of using kubectl against different clusters (local vs remote).

```
# Use this command to list k8s contexts.
# If you've started a minikube cluster, you'll see a minikube context name.
# This command also lists other contexts like remote GKE clusters.
kubectl config get-contexts

# Switch kubectl to minikube
kubectl config use-context [context name for minikube]

# Switch kubectl to remote GKE cluster.
gcloud container clusters get-credentials [remote cluster name]
```